### DISCUSSION TITLE
Helper/Instructor recruitment and skill development

### DISCUSSION THREAD

**Discussion by karenword** [carpentries/trainers#118] [Local Communities]
How can local communities recruit new people to become Helpers and Instructors? How can communities support them in continuation, advancement, and development of skills that have value **to them** in the context of these roles?

**Comment by karenword** [comment]
This question was addressed at a Trainers meeting on June 17, 2021. This is a summary of meeting responses.

**Who to recruit**
* enthusiastic workshop trainees/"unofficial helpers" -> Helpers -> Instructors
* library staff
* research software engineers 
* PhD students
* post-docs
* colleagues
* partner communities (e.g. R/Python groups)

**How to recruit**
* pay attention to promising/helpful trainees in workshops and invite them personally
* collect & share learner testimonials about value of (technical) workshops
* word-of-mouth / Instructors get excited about The Carpentries and promote to colleagues
* widely distributed Instructor Training application can generate new applicants beyond target/base cohort
* add workshop trainees to mailing list (if allowed) to inform them of future opportunities
* value all contributions (big and small) / ease in to bigger commitments as they get comfortable
* when recruiting from groups, can incentivize them to send helpers by offering seats in workshops.
* advertise relevant benefits (e.g. teaching opportunity for post-docs on the hunt for teaching jobs)
* convince higher-ups to make training a job duty to recruit staff
* (platinum model): offer Instructor training widely, recruit most enthusiastic trainees to actually teach

**How to support**
* monthly Instructor development meeting, sessions e.g. teaching debugging, stereotype threat
* debriefs after every workshop 
* study groups / book groups about teaching
* teaching to talk about service activities in ways that promote career advancement e.g. in annual review
* charge a small amount for workshops; use most to cover lunch etc but reserve remainder to create a travel fund for conference/CarpentryCon
* practice/feedback sessions


**Challenges**
* it can take some time (years) to build momentum, especially for recruiting workshop trainees 
* silos (especially when organizing from a non-central point e.g. academic department vs. library)
* recruiting people not otherwise interested/motivated can take some convincing (e.g. RSEs), must fit with personal/professional goals and contribute to career advancement.
* PhD students are ephemeral -- moreso in some places than others (e.g. 3 year degree programs vs 4+ year averages)
* time and effort, both for organizers and Instructors/Helpers
* lack of intermediate content can make it hard to retain trainees in community, especially if intermediate training is available elsewhere.
* shift in mindset from learner to contributing community member is difficult to achieve
* membership is necessary; institutional support is unpredictable; incentives change after a few years of membership
* recent trainees sometimes need help addressing questions outside their knowledge base
* some things just don't work. Persistence is essential.

**Reply by dvanic** [reply]
I think "must fit with personal/professional goals and contribute to career advancement", with clear paths for where one can go as an instructor/maintainer, is pivotal. 
Perhaps we should have Instructor stories somewhere on the Carpentries blog/site, to showcase some of the ways being part of the Carpentries has helped us find new roles and grow in existing ones?

**Reply by brownsarahm** [reply]
After [Belinda Weaver](https://carpentries.org/blog/2018/09/my-carpentries-pathway/) wrote about her pathway, it seemed like there was a plan to make it a series, but I can't find additional posts, though [form to submit one](https://carpentries.typeform.com/to/akjfDR) is still open.


maybe we:

1. make those more findable for new instructors, as a collection
2. as trainers, contribute posts
3. as leaders of local communities, encourage others to contribute more posts

**Reply by dvanic** [reply]
Great - I completely forgot that!

I think we should expand on 

> 1. make those more findable for new instructors, as a collection

as well, in that we make sure they go out in Omar's mailouts?
